### DISCUSSION TITLE
Forcing a GPU to run on?

### DISCUSSION THREAD

**Discussion by HorreC** [LostRuins/koboldcpp#356] [General]
When I load it always wants to run on my workstation card not the 7900xtx.
It shows both 

```
Platform:0 Device:0  - AMD Accelerated Parallel Processing with gfx1012:xnack-
Platform:0 Device:1  - AMD Accelerated Parallel Processing with gfx1100
```

But I cant find a way to force the device.
`
ggml_opencl: selecting device: 'gfx1012:xnack-'`

is what I get no matter what I try, am I missing something?

**Comment by LostRuins** [comment]
To use the first device, run `--useclblast 0 0`
To use the second device, run `--useclblast 0 1`

**Reply by HorreC** [reply]
this is great, thank you for such a quick answer.
